tests/libstorage/pv.go

```diff
@@ -0,0 +1,54 @@
+package libstorage
+
+import (
+	"context"
+
+	k8sv1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+
+	"kubevirt.io/kubevirt/pkg/apimachinery/patch"
+	"kubevirt.io/kubevirt/tests/framework/kubevirt"
+)
+
+func isLocalPV(pv k8sv1.PersistentVolume) bool {
+	return pv.Spec.NodeAffinity != nil &&
+		pv.Spec.NodeAffinity.Required != nil &&
+		len(pv.Spec.NodeAffinity.Required.NodeSelectorTerms) > 0 &&
+		(pv.Spec.VolumeMode == nil || *pv.Spec.VolumeMode != k8sv1.PersistentVolumeBlock)
+}
+
+func isPVAvailable(pv k8sv1.PersistentVolume) bool {
+	return pv.Spec.ClaimRef == nil
+}
+
+func MakePVAvailable(ctx context.Context, pv *k8sv1.PersistentVolume) error {
```

Comment:
So we could get rid of this if we use pv reclaim policy "recycle" https://kubernetes.io/docs/tasks/administer-cluster/change-pv-reclaim-policy/ https://github.com/kubevirt/kubevirt/blob/31fa6dda29259ef93272e1b7caecc731fcf8541a/automation/test.sh#L366